### DISCUSSION TITLE
How to show dropped connection?

### DISCUSSION THREAD

**Discussion by amigthea** [evilsocket/opensnitch#1216] [Q&A]
I cannot find a way to show the dropped connection, not even in the debug logs

![image](https://github.com/user-attachments/assets/82331a20-a234-40ab-8338-090e3dc845c9)

**Comment by gustavo-iniguez-goya** [comment]
Hi @amigthea ,

The dropped connections are the ones we denied or rejected. From the GUI you can filter by Reject or Deny.

In the logs it's printed as:
` DBG  ✘ /usr/lib/firefox-esr/firefox-esr -> 39896:fe80::b5cd:ba:a223:5b0a => geo.yahoo.com (fe80::1):53, mark: 0 (000-block-domains)`
